### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by Sandesh-03** [cheesecakeufo/komorebi#332]
komorebi is not opening

**Comment by mohsnMohsni** [comment]
i have same problem on ubuntu 22.04

**Comment by Gimas-aza** [comment]
similar to fedora 36

**Comment by ShadmanAhm3d** [comment]
yep same problem here bois , days of komorebi are coming to end slowly + cant believe we  still dont have an app that allows live wallpaper in gnome.

**Comment by ZA0068** [comment]
me too with Linux Mint 20.2

**Comment by NoTooth666** [comment]
I have this exact issue with Ubuntu 20.04.1
Installs fine just won't open not matter what i do
